src/server.js

```diff
@@ -148,6 +147,25 @@ export const addRoutes = (server, stylesheetsPath) => {
   const routes = getRouteDefinitions();
   routes.forEach((route) => server.route(route));
 };
+
+/**
+ * Adds a middleware function to the server that adds a custom HTTP response header, `X-App-Version`, to all
+ * responses with the value of the `APP_VERSION` environment variable or a default value of `1.0.0` if the variable
+ * is not set.
+ * @param {object} server - The server instance on which to add the middleware.
+ */
+export const addMiddleware = (server) => {
+  server.ext('onPreResponse', (request, h) => {
+    const response = request.response;
+    if (response.isBoom) {
+      response.output.headers['X-App-Version'] = pkg.version || '1.0.0';
```

Comment:
Could we change the default to unset or similar? Just so that we don't imply that there is a deployed version when something is wrong.

Reply:
Amended.